api/v1beta1/event.proto

```diff
@@ -551,9 +551,7 @@ enum EventId {
     security_task_set_rlimit = 1086;
     security_settime64 = 1087;
     chmod_common = 1088;
-    open_file_ns = 1089;
-    open_file_mount = 1090;
-    security_task_prctl = 1091;
+    security_task_prctl = 1089;
```

Comment:
Does this event id match the bpf code?
It worth sanity testing the last few events with the changes in this PR

Reply:
Did run another check and all seem synced. All non internal in go is already in proto. Their ids aren't aligned (as shouldn't) but they're in correct order.